rocky/rocky/signals.py

```diff
@@ -22,6 +25,12 @@
 # Signal sent when a user logs in
 @receiver(user_logged_in)
 def user_logged_in_callback(sender, request, user, **kwargs):
+    create_missing_groups()
+    logger.info("Groups ", username=user.get_username())
```

Comment:
Looks like the info logs texts are now in the wrong method

Reply:
Not sure what happened here, these lines are obviously mixed up and belong in the new make_groups listener. Fixed!